### ISSUE TITLE
Rspamd : "monitored errors"

### ISSUE THREAD

**Issue by Vickylatruite** [Mailu/Mailu#206]
`docker logs` on rspamd container shows tons of lines like this one:

`<xxx>; monitored; rspamd_monitored_dns_cb: DNS reply returned 'no error' for multi.uribl.com while 'no records with this name' was expected
`
This error means that the public DNS resolver I'm using (OVH in my case, but Google DNS is also concerned) is blocked on uribl.com.

Source : [https://rspamd.com/doc/faq.html#why-do-i-have-monitored-errors-in-my-log-files](https://rspamd.com/doc/faq.html#why-do-i-have-monitored-errors-in-my-log-files)

It should be nice to have some flexibility to enable or not these services.

See: [https://rspamd.com/doc/modules/surbl.html](https://rspamd.com/doc/modules/surbl.html)

**Comment by Vickylatruite** [comment]
Had kind of same issue with `list.dnswl.org`, that is a RBL module (See: [https://rspamd.com/doc/modules/rbl.html](https://rspamd.com/doc/modules/rbl.html))

Workaround: create a volume for rspamd container linked to /etc/rspamd/local.d folder. This folder can contains conf files that overrides default ones. 

Source: [https://rspamd.com/doc/modules/surbl.html#disabling-surbls](https://rspamd.com/doc/modules/surbl.html#disabling-surbls)

**Comment by kaiyou** [comment]
These were disabled from the default Rspamd setup. However, it is now possible to override Rspamd configuration directly in `/mailu/overrides/rspamd` :)
